injected/integration-test/duckplayer.spec.js

```diff
@@ -39,7 +39,7 @@ test.describe('Thumbnail Overlays', () => {
         const navigation = overlays.requestWillFail()
         await overlays.clicksFirstShortsThumbnail()
         const url = await navigation
-        await overlays.opensShort(url)
+        overlays.opensShort(url)
```

Comment:
This changes the meaning of the test. I believe we're intending to validate if this method throws which won't happen without an `await`.

Reply:
Ignore, checked the fn signature